app/views/days/index.html.erb

```diff
@@ -1,78 +1,80 @@
-<script src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js" type="text/javascript"></script>
-<script src="https://cdn.jsdelivr.net/gh/alumuko/vanilla-datetimerange-picker@latest/dist/vanilla-datetimerange-picker.js"></script>
-
-<head>
-<%= javascript_importmap_tags %>
-</head>
-
-<p style="color: green"><%= notice %></p>
-
-<pre class="d" id="t1">
-    __  ___                ________                __            
-   /  |/  /___  ____  ____/ /_  __/________ ______/ /_____  _____
-  / /|_/ / __ \/ __ \/ __  / / / / ___/ __ `/ ___/ //_/ _ \/ ___/
- / /  / / /_/ / /_/ / /_/ / / / / / /  / /_/ / /__/ ,< /  __/     
-/_/  /_/\____/\____/\__,_/ /_/ /_/   \__,_/\___/_/|_|\___/_/     
-                                                                 
-</pre>
-
-<body id="body">
-
-  <button type='button' class= "backbutton" onclick="location.href='<%= new_day_path() %>';">Create new daily entry</button> 
-
-  <div id="piechartbutton" class="leftshift1" style="margin-bottom: 1%;"><button onclick="switchpiechart()">See pie chart</button></div>
-  <div id="linechartbutton" class="hidden leftshift1" style="margin-bottom: 1%;"><button onclick="switchlinechart()">See line chart</button></div>
-  <div id="columnchartbutton" class="hidden leftshift1" style="margin-bottom: 1%;"><button onclick="switchcolumnchart()">See column chart</button></div>
-
-  <div id="linechartbuttons">
-    <div style="display: flex; margin-left: 2%;" >
-      <div class="leftshift1"><button id="linechartbuttonA" class="buttonselect" onclick="switchlinechartall()">All</button></div>
-      <div class="leftshift1"><button id="linechartbuttonC" onclick="switchcustom()">Custom range</button></div>
-      <div class="leftshift1 hidden" id="calendersearchbutton">
-        <%= form_with url: days_path, method: :get do |form| %>
-          <%= form.text_field :days_between, value: params[:days_between], class: "d", data: {controller: "daterangepicker"} %>
-          <%= form.submit "search", class: "d" %>
-        <% end %>
+<% if current_user %>
```

Comment:
remove the <%if current_user %> and the  <%end%> this should be handled by the controller instead